### ISSUE TITLE
Support for LLVM 14.0.0

### ISSUE THREAD

**Issue by NaderAlAwar** [ChrisCummins/ProGraML#188]
## 🚀 Feature

Add support for LLVM 14.0.0 IR.

## Pitch

Is support for newer versions of LLVM in the works? If not, I would be interested in working on it myself, although I'm not sure where to start.

**Comment by ChrisCummins** [comment]
Hi @NaderAlAwar, good question! Adding support for a new LLVM version is totally do-able, and patches very welcome :) However, I'm not sure if we can add support for LLVM-14 yet. We compile against the [official LLVM builds](https://github.com/llvm/llvm-project/tags) and I don't think there has been a release for 14 yet.

Cheers,
Chris

**Comment by NaderAlAwar** [comment]
I ended up forking this repo and https://github.com/ChrisCummins/bazel_llvm at https://github.com/NaderAlAwar/ProGraML/ and https://github.com/NaderAlAwar/bazel_llvm. I was able to add support for the latest release (13).

Do you want me to submit a PR to the bazel_llvm repo first, and then submit a PR here?

**Comment by ChrisCummins** [comment]
> Do you want me to submit a PR to the bazel_llvm repo first, and then submit a PR here?

That would be ideal, thanks! In the meantime you can point [the bazel_llvm dep used by ProGraML](https://github.com/ChrisCummins/ProGraML/blob/development/tools/bzl/deps.bzl#L23-L30) to your forked version to check it works.

Cheers,
Chris

**Comment by NaderAlAwar** [comment]
Yeah I was able to install it with my forked repo. PR to other repo is here ChrisCummins/bazel_llvm#6
